### ISSUE TITLE
Properly handle namespaced Workers

### ISSUE THREAD

**Issue by felixbuenemann** [davydovanton/sidekiq-statistic#46]
The plugin currently does not properly handle worker classes, which are nested in a module, eg. `Foo::BarWorker` and `Foo::BazWorker` wil display as a single entry `Foo` with wrong stats in the WorkersTable.

My guess is that this is because `:` is also used as the separator for the redis keys (eg. `my_app_production:sidekiq:2015-08-19:Foo::BarWorker:timeslist`.

**Comment by felixbuenemann** [comment]
The problem seems to be in `Sidekiq::Statistic::Base#get_statistic_hash` and `Sidekiq::Statistic::Realtime.realtime_hash`because of how the keys are split.

Maybe you can do the following:
```ruby
class Sidekiq
  module Statistic
    class Base
      KEY_SEPARATOR = /(?<!:):(?!:)/.freeze
    private
      def get_statistic_hash(conn, redis_hash)
        conn
          .hgetall(REDIS_HASH)
          .each do |keys, value|
            *keys, last = keys.split(KEY_SEPARATOR)
            keys.inject(redis_hash, &key_or_empty_hash)[last.to_sym] = to_number(value)
          end
      end
    end
  end
end
```

**Comment by sancopanco** [comment]
pull request #53  may fix this

**Comment by felixbuenemann** [comment]
@sancopanco I've already got PR #48 to fix this.

**Comment by davydovanton** [comment]
closed by #48
